experimental/Matrix/matrix.g

```diff
@@ -0,0 +1,139 @@
+BindGlobal("HelloWorld", function()
+    return "Hello World!";
+end);
+
+BindGlobal("JuliaMatrixFamily", NewFamily("JuliaMatrixFamily"));
+
+DeclareRepresentation(
+  "IsJuliaMatrixRep",
+  IsComponentObjectRep and IsAttributeStoringRep and IsMatrixObj);
+  
+JuliaMatrixIdentificationType :=
+    NewType( JuliaMatrixFamily, IsJuliaMatrixRep);
```

Comment:
I think you need to add `IsMutable` here in order to allow for mutable instances
```suggestion
BindGlobal("JuliaMatrixIdentificationType", NewType(JuliaMatrixFamily, IsJuliaMatrixRep and IsMutable));
```